pyedflib/edfwriter.py

```diff
@@ -448,20 +453,21 @@ def setGender(self, gender):
         self.gender = gender2int(gender)
         self.update_header()
 
-    def setDatarecordDuration(self, duration):
+    def setDatarecordDuration(self, record_duration):
         """
-        Sets the datarecord duration. The default value is 100000 which is 1 second.
-        ATTENTION: the argument "duration" is expressed in units of 10 microSeconds!
-        So, if you want to set the datarecord duration to 0.1 second, you must give
-        the argument "duration" a value of "10000".
-        This function is optional, normally you don't need to change
-        the default value. The datarecord duration must be in the range 0.001 to 60  seconds.
+        Sets the datarecord duration. The default value is 1 second.
+        The datarecord duration must be in the range 0.001 to 60  seconds.
+        Usually, the datarecord duration is calculated automatically to
+        ensure that all sample frequencies are representable, nevertheless,
+        you can overwrite the datarecord duration manually. This can, however,
+        lead to unexpected side-effects in the sample frequency calculations.
+        
         Returns 0 on success, otherwise -1.
 
         Parameters
         ----------
-        duration : integer
-            Sets the datarecord duration in units of 10 microSeconds
+        record_duration : integer
+            Sets the datarecord duration in units of seconds
```

Comment:
This was wrong. We insert the `record_duration` in seconds, not 10mS (this is how it is saved later inside the edf, but not how we indicate it here)